slack.json

```diff
@@ -0,0 +1,21 @@
+{
+  "runtime": "deno",
+  "hooks": {
+    "build": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-write --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts",
+    "get-manifest": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts --manifest",
```

Comment:
@shapirone @filmaj - Connecting the dots here, even if this line is not included (in favor of being implemented by `get-hooks` per the deno-slack-hooks repo, can we align on `get-manifest` as the new name? It is standardized with `get-hooks`.

Reply:
Sure. We need to update the CLI repo and the hooks repo in the meantime, and likely the builder repo too until the hooks repo is public. Can we update the CLI/SDK spec with all these changes too?

Reply:
* CLI (possibly this already is in @sopokuagyemang's sights!) 
* CLI / SDK spec - ✅ 
* Hooks repo - can I help with either of these?
* Builder repo - can I help with either of these?

Reply:
Sure, how about you send a PR to the hooks repo and I'll send one for the builder?

Reply:
@filmaj - Here you go! https://github.com/slackapi/deno-slack-hooks/pull/3